packages/@ember/debug/lib/deprecate.ts

```diff
@@ -14,9 +14,12 @@ export interface DeprecationOptions {
   id: string;
   until: string;
   url?: string;
+  for: string;
+  since: object;
```

Comment:
I think we want this to be:

```ts
export type DeprecationStages = 'available' | 'enabled';
export interface DeprecateOptions {
  // ...snip...
  since: Record<DeprecationStages, string>;
}
```